tedge/src/cli/disconnect/disconnect_bridge.rs

```diff
@@ -70,6 +70,19 @@ impl DisconnectBridgeCommand {
             self.service_manager()
                 .stop_and_disable_service(self.cloud.dependent_mapper_service(), std::io::stdout());
         }
+        match self.cloud {
+            Cloud::C8y => {
+                if which("tedge_agent").is_ok() && which("tedge_mapper").is_ok() {
+                    self.service_manager().stop_and_disable_service(
+                        SystemService::TEdgeSMMapperC8Y,
+                        std::io::stdout(),
+                    );
+                    self.service_manager()
+                        .stop_and_disable_service(SystemService::TEdgeSMAgent, std::io::stdout());
+                }
+            }
+            _ => todo!(),
```

Comment:
This MUST be removed.

* `todo!()` will panic on `tedge disconnect az`.
* Simply do no nothing since there is no sm-az mapper yet.

Reply:
fixed